### ISSUE TITLE
Self Discovery?

### ISSUE THREAD

**Issue by sabzo** [sabzo/ayanda#9]
While looking for a service should the device also make its presence known? In the case of LAN: Announcing during discovery will ensure that the device discovers itself. 

Maybe device can identify itself and remove itself from devices found..? 
Thoughts? @n8fr8

**Comment by n8fr8** [comment]
Yes, agreed, that generating an ID and using the text record to check it, makes sense.
